scripts/update/radarr.sh

```diff
@@ -65,91 +65,91 @@ if [[ -f /install/.radarr.lock ]]; then
     ### Populate swizdb values to defaults if empty; these must be kept in sync with install
     #ToDo: Fix this if logic; we want to check if the DB has something; if so, do nothing.. if not. populate DB with defaults
     app_name="radarr"
-    if ! "$(swizdb get $app_name/app_name)"; then
+    if [ 1 = "$(swizdb get $app_name/app_name)" ]; then
```

Comment:
bruh wot?

Reply:
a very poor attempted at leveraging the response

>swizdb get $key
>Returns the content of $key
> Returns exit code 1 in case the key is not present

Reply:
https://shapeshed.com/unix-exit-codes/

https://discord.com/channels/577667871727943696/702953563340537896/837411125603729449